gtsam/constraint/NonlinearInequalityConstraint.cpp

```diff
@@ -0,0 +1,178 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file    NonlinearInequalityConstraint.cpp
+ * @brief   Nonlinear inequality constraints in constrained optimization.
+ * @author  Yetong Zhang, Frank Dellaert
+ * @date    Aug 3, 2024
+ */
+
+#include <gtsam/constraint/NonlinearInequalityConstraint.h>
+
```

Comment:
Include "
#include <gtsam/inference/FactorGraph-inst.h>"

Reply:
Added this line, CI still reports the same error in Windows environments.